server/ringhash/ringhash.go

```diff
@@ -4,6 +4,7 @@ package ringhash
 
 import (
 	"encoding/ascii85"
+	"hash/crc32"
 	"hash/fnv"
```

Comment:
Remove

Reply:
FNV is still used for generating the ring signature.
I'll revisit it once I pick a good murmurhash candidate.